spec/acceptance/wifi_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper_acceptance'
+
+describe 'wifi class' do
+  it 'ensure network-manager is installed' do
+    pp = <<~MANIFEST
+    package { 'network-manager':
+      ensure => installed,
+    }
+    MANIFEST
+
+    apply_manifest(pp, catch_failures: true)
+  end
+
+  it 'configures Wi-Fi access points' do
+    pp = <<~MANIFEST
+    # https://www.youtube.com/watch?v=bLE7zsJk4AI
+
+    wifi::infrastructure { 'Rocketjump GGGGG':
```

Comment:
If I run locally something like this (ie. heredoc), `rubocop` asks me to ident like this:
```ruby
puppet = <<~PUPPET
  Hello Puppet
PUPPET
```

Where these ones are bad:
```ruby
puppet = <<~PUPPET
  Hello Puppet
  PUPPET
```
```ruby
puppet = <<~PUPPET
Hello Puppet
PUPPET
```

Reply:
Not worse… Might be a good candidate to add to modulesync_config?  Needs a rubocop version bump maybe?